README.md

```diff
@@ -1,12 +1,25 @@
-# Journald
-![template](https://github.com/linux-system-roles/template/workflows/tox/badge.svg)
+# Journald | [![Ansible Lint][ansible-lint-status]][ansible-lint] [![Ansible Plugin Scan][ansible-scan-status]][ansible-scan] [![Ansible Test][ansible-test-status]][ansible-test] [![Weekly CI trigger][weekly-test-status]][weekly-test]
+
+[ansible-lint]: https://github.com/linux-system-roles/journald/actions/workflows/ansible-lint.yml
+[ansible-lint-status]: https://github.com/linux-system-roles/journald/actions/workflows/ansible-lint.yml/badge.svg
+
+[ansible-scan]: https://github.com/linux-system-roles/journald/actions/workflows/ansible-plugin-scan.yml
+[ansible-scan-status]: https://github.com/linux-system-roles/journald/actions/workflows/ansible-plugin-scan.yml/badge.svg
+
+[ansible-test]: https://github.com/linux-system-roles/journald/actions/workflows/ansible-test.yml
+[ansible-test-status]: https://github.com/linux-system-roles/journald/actions/workflows/ansible-test.yml/badge.svg
+
+[weekly-test]: https://github.com/linux-system-roles/journald/actions/workflows/weekly_ci.yml
+[weekly-test-status]: https://github.com/linux-system-roles/journald/actions/workflows/weekly_ci.yml/badge.svg
 
 This role provides an easy way to configure systemd-journald logging service.
 
 ## Requirements
+
```

Comment:
Is this something that would be noted by markdownlint?

Reply:
Yes, it would be.

_MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines markdownlint[MD022](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md022.md)_

Reply:
ok - I'd like to add the markdownlint github action to all of the system roles repos - but then that will be a *lot* of work for someone to clean up all of our docs . . . https://github.com/linux-system-roles/.github/blob/main/playbooks/templates/.github/workflows/markdownlint.yml